components/datepicker/overview.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Overview
+title: Blazor DatePicker Overview | Telerik UI for Blazor
 page_title: Date Picker Overview
```

Comment:
This is the SEO title that you could change. However, note the following:
- This title is generated by a template that looks like this: "Blazor" + page_title + - Telerik UI for Blazor.
- The currently rendered title is "Blazor Date Picker Overview - Telerik UI for Blazor".
- There is only one thing that you could change and this is the component name: Date Picker -> DatePicker.
- Changing the separator (from `-` to `|`) is not an option as it is part of the template and we have an agreement with the SEO team that we won't change the separators. If they've requested this change, this is probably a mistake on their part.

```suggestion
page_title: DatePicker Overview
```